commit 8520282f7533d76b6c38e041d2f46898140021ff
Author: Matthias Clasen <mclasen@redhat.com>
Date:   Mon Mar 3 23:30:55 2014 -0500

    Fix up builder test
    
    Dialogs now report two children. Thats alright, adapt
    the test to deal with it.

diff --git a/testsuite/gtk/builder.c b/testsuite/gtk/builder.c
index f0f1a14..43a0441 100644
--- a/testsuite/gtk/builder.c
+++ b/testsuite/gtk/builder.c
@@ -1028,7 +1028,7 @@ test_children (void)
   g_assert (dialog != NULL);
   g_assert (GTK_IS_DIALOG (dialog));
   children = gtk_container_get_children (GTK_CONTAINER (dialog));
-  g_assert (g_list_length (children) == 1);
+  g_assert (g_list_length (children) == 2);
   g_list_free (children);
   
   vbox = gtk_builder_get_object (builder, "dialog1-vbox");
